### ISSUE TITLE
configurable parameter for SDR selection

### ISSUE THREAD

**Issue by mariobaldini** [mariobaldini/gnuradio-gfsk-codec#3]
Pass as one argument the SDR type (ie. ettus or pluto), that will be internally utilized to adjust the gnuradio flow (if needed).

This eliminates the need of multiple/duplicaded GR flows for different SDRs (with hardcoded parameters)

**Comment by Theis159** [comment]
This seems to be tricky with Pluto. Right now to do that we would need to have two different sources and sinks (PlutoSDR source/sink and osmocom source/sink) and this does not allow us to run the code without having both devices connected. There is a not validated repository of osmocom source/sink that accepts PlutoSDR as device, need to look further into it.

**Comment by Theis159** [comment]
I have looked further into the osmocom documentation: as for now to be able to this we need to build the gnuradio repository. This issue will not be solved until the osmocom source/sink have a reliable support to PlutoSDR

**Comment by mariobaldini** [comment]
The feature itself exists. The wontfix labels implies that it was not implemented. 

The original intent was to show that 'wontfix' for the PlutoSDR? 

Currently is it possible to select by argument multiple SDRs? eg. ettus or funcube?

**Comment by mariobaldini** [comment]
Currently the codec fully supports RTLSDR (including Funcube) and Ettus?
If yes and it's possible to easily change devices from the arg line, I will mark as solved (even without PlutoSDR -- since the issue/requirement is not about pluto itself)

**Comment by Theis159** [comment]
Yes, you can mark as solved. The only non supported device (from the SDRs we currently have at CSU) is the PlutoSDR. I will add all the devices supported for this flow and how to set them from the arg line in the documentation
